### ISSUE TITLE
[beta] IX-F importer: IX "Last Updated" timestamp is based on IX-F import time - not sure that is intended

### ISSUE THREAD

**Issue by ccaputo** [peeringdb/peeringdb#812]
The "Last Updated" timestamp of an IX is updated by the IX-F import time of a IX-F JSON URL provided by the IX.  The problem is that this update time then affects the "Most Recent Updates" we show on the main home page.  Thus whenever the IX-F Importer cron job runs, the home page "Most Recent Updates" list for IXes will essentially become useless.

For example, due to the importer run on beta, we see the following:

![Screen Shot 2020-08-12 at 11 56 20 AM](https://user-images.githubusercontent.com/1417833/90055863-e0054880-dc92-11ea-8317-3d3a397e33d9.png)

Those IXes didn't actually log in and make an update.  They simply are the highest numbered ix_ids, in reverse order, that had their IX-F JSONs processed by the importer last.

I recommend that "Last Updated" timestamp of an IX _not_ be updated by the IX-F Importer.  @mcmanuss8 and @peeringdb/pc, thoughts?

**Comment by mcmanuss8** [comment]
I agree. Let's fix this, ideally before the August release goes out to avoid confusion

**Comment by arnoldnipper** [comment]
+1 and moved to 3a

**Comment by Yo-Robinson** [comment]
Makes sense +1

**Comment by shane-kerr** [comment]
+1

**Comment by arnoldnipper** [comment]
@vegu, why is the timestamp updated? We don't take any information from the IX-F JSON other than from member_list. Or do we?

**Comment by vegu** [comment]
There are some ix-f related properties that get updated during an import run.

- `ixf_net_count`
- `ixf_last_import`

If neither of those change, the exchanges shouldn't receive a change to their "updated" field during the import, so once the initial import is done it wouldn't affect all exchanges like this.

We can also make those updates without affecting the `updated` field if that's the preference.

**Comment by ccaputo** [comment]
@vegu, reviewing, as an example:

```
MySQL [peeringdb]> SELECT name,updated,ixf_last_import FROM peeringdb_ix WHERE id=13 OR id=3226;
+-------------+---------------------+----------------------------+
| name        | updated             | ixf_last_import            |
+-------------+---------------------+----------------------------+
| SIX Seattle | 2020-08-11 18:25:37 | 2020-08-11 18:25:26.897236 |
| IX42 LAS    | 2020-08-14 00:15:11 | 2020-08-14 00:15:09.945838 |
+-------------+---------------------+----------------------------+
2 rows in set (0.00 sec)
```

and the IX-F URLs for each, has left me confused.  Right now, the beta import is running daily as follows:

```
00 00 * * * chronic flock -x -w 240 /tmp/pdb_process_admin_tool_command.lock /home/pdb/bin/pdb-container pdb_ixf_ixp_member_import --commit
```

And the timestamps of both of these URLs are updating every minute or so, ex:

SIX:

```
"timestamp":"2020-08-14T16:49:02Z",
```

IX42 LAS:

```
"timestamp": "2020-08-14T16:49:29Z",
```

and yet at the most recent 0000 UTC run, only IX42 got an updated `ixf_last_import` timestamp.  Is that because `ixf_last_import` only changes if the import content (other than timestamp) results in a change?  In that case, I am not sure `ixf_last_import` is accurately named, or maybe this is another bug, since I think it makes sense to have `ixf_last_import` reflect the last time an IX-F JSON URL was processed, even if there is no new data other than an updated timestamp.

Either way, yes, please make it so an IX-F import does not change the IX `updated` field regardless of whether `ixf_net_count` or `ixf_last_import` change.  It only makes sense to change the `updated` field if public information on the related /ix/ page is affected.  Thanks.

**Comment by vegu** [comment]
@ccaputo 

> In that case, I am not sure ixf_last_import is accurately named, or maybe this is another bug, since I think it makes sense to have ixf_last_import reflect the last time an IX-F JSON URL was processed, even if there is no new data other than an updated timestamp.

Correct, it only updates on an import that successfully ran with changes done.

Looks like the `ixf_last_import` field was implemented as part of the `ixf_net_count` field in #683  and the closest i can find to an actual spec for it is this comment: https://github.com/peeringdb/peeringdb/issues/683#issuecomment-615462575 

> Either way, yes, please make it so an IX-F import does not change the IX updated field regardless of whether ixf_net_count or ixf_last_import change. It only makes sense to change the updated field if public information on the related /ix/ page is affected. Thanks.

Will do

**Comment by ccaputo** [comment]
> @ccaputo
> 
> > In that case, I am not sure ixf_last_import is accurately named, or maybe this is another bug, since I think it makes sense to have ixf_last_import reflect the last time an IX-F JSON URL was processed, even if there is no new data other than an updated timestamp.
> 
> Correct, it only updates on an import that successfully ran with changes done.
> 
> Looks like the `ixf_last_import` field was implemented as part of the `ixf_net_count` field in #683 and the closest i can find to an actual spec for it is this comment: [#683 (comment)](https://github.com/peeringdb/peeringdb/issues/683#issuecomment-615462575)

Cool.  I interpret that comment to mean last import, with or without changes.  Please change `ixf_last_import` to reflect the last time the importer was successfully run for this IX, rather than the last time there were importer inspired changes.

> > Either way, yes, please make it so an IX-F import does not change the IX updated field regardless of whether ixf_net_count or ixf_last_import change. It only makes sense to change the updated field if public information on the related /ix/ page is affected. Thanks.
> 
> Will do

Thanks.

**Comment by ccaputo** [comment]
Fix confirmed (2.21.0.6).
